### PR TITLE
Fix to return _id to when it is model id

### PR DESCRIPTION
**Pull request by virkt25**
Fixes an issue where if _id was set as id on model, any query won’t
return _id anymore.

### CODE REVIEW

File: test/id.test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      _id: {type: String, id: true},
+    }, {forceId: true});
+
+    Task.destroyAll(function(err) {
+      done();
+    });
+  });
+
+  after(function(done) {
+    Task.destroyAll(function() {
```
Comment (ssh24) [inline test/id.test.js]: `Task.destroyAll(done)`

File: test/id.test.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  before(function(done) {
+    db = getDataSource();
+
+    Task = db.define('Task', {
+      task: {type: String},
+      owner: {type: String},
+      _id: {type: String, id: true},
+    }, {forceId: true});
+
+    Task.destroyAll(function(err) {
```
Comment (ssh24) [inline test/id.test.js]: Why are you doing a destroyAll here?

Comment (b-admike) [inline test/id.test.js]: Do we need this call for `destroyAll()` here?

**Comment by virkt25** [comment]
connect to #165

File: test/id.test.js (hunk 2 above)
Comment (virkt25) [inline test/id.test.js]: Changed to automigrate.

Comment (virkt25) [inline test/id.test.js]: Changed to automigrate.

File: test/id.test.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  });
+
+  it('result should include _id', function(done) {
+    var ownerName = 'John Smith';
+    Task.create({
+      task: 'some random task',
+      owner: ownerName,
+    }, function(err, task) {
+      if (err) return done(err);
+      var taskId = task._id;
```
Comment (ssh24) [inline test/id.test.js]: I would assert for `task` and `task._id` first.
```js
should.exist(task);
should.exist(task._id);
```

**Comment by kjdelisle** [comment]
+ 1 to @ssh24's remaining suggestion; after that, LGTM.

**Comment by kjdelisle** [comment]
Node 0.12 and 0.12 support needs further discussion, but it's not a real blocker for this fix.

**Comment by kjdelisle** [comment]
All relevant tests were passing, ignoring the above.

**Pull request merged**